src/main/scala/higherkindness/rules_scala/workers/common/FileUtil.scala

```diff
@@ -59,6 +59,29 @@ object FileUtil {
     path.getFileName().toString().stripPrefix("header_").stripPrefix("processed_")
   }
 
+  /**
+   * Given a Bazel path to the bazel output directory, return a path excluding the bazel configuration specific parts of
+   * the path. This is analogous to Bazel's File.short_path function. Fair warning: this function is super good enough,
+   * but is likely not perfect.
+   */
+  def bazelShortPath(path: Path, replaceExternal: Boolean = true): Path = {
+    val nameCount = path.getNameCount()
+    val pathString = path.toAbsolutePath().normalize().toString()
```

Comment:
Is this ever used?

Reply:
Nope. It's a remnant of another approach. I'll remove.